datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -295,21 +299,39 @@ def __call__(sshurl, target=None, target_dir=None,
                             "You will not be able to push to checked out "
                             "branch. Error: %s", path, exc_str(e))
 
-            # enable post-update hook:
+            # create post_update hook to refresh dataset metadata on publication server
+            lgr.info("Enabling git post-update hook ...")
```

Comment:
hm, how come this code is claimed to be not tested... I thought there is at least a smoke test using ssh.  please check/fix